src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -932,6 +943,7 @@ private function createOrders(
             $this->addVariantWithPriceToOrder($order, $sampleProductVariant, $price);
 
             if ($isFulfilled) {
+                $order->setPaymentState(OrderPaymentStates::STATE_PAID);
                 $this->applyTransitionOnOrder($order, OrderTransitions::TRANSITION_FULFILL);
```

Comment:
Should we consider a guard for payment and shipment state on order? /cc @Sylius/core-team

Reply:
Agreed, "fulfill" transition should never be called explicitly 🐎